diff -Nur -x '*.orig' -x '*.rej' e-smith-tinydns-1.0.0/root/etc/e-smith/db/configuration/migrate/tinydns_ListenIP mezzanine_patched_e-smith-tinydns-1.0.0/root/etc/e-smith/db/configuration/migrate/tinydns_ListenIP
--- e-smith-tinydns-1.0.0/root/etc/e-smith/db/configuration/migrate/tinydns_ListenIP	1969-12-31 19:00:00.000000000 -0500
+++ mezzanine_patched_e-smith-tinydns-1.0.0/root/etc/e-smith/db/configuration/migrate/tinydns_ListenIP	2006-09-14 15:28:45.000000000 -0400
@@ -0,0 +1,11 @@
+{
+    # $tinydns{ListenIP} must not be set in serveronly mode
+    my $tinydns = $DB->get('tinydns');
+    return unless $tinydns;
+    my $mode = $DB->get_value('SystemMode');
+    return unless $mode;
+    if ($mode eq 'serveronly')
+    {
+	$tinydns->delete_prop("ListenIP");
+    }
+}
