diff -Nur -x '*.orig' -x '*.rej' e-smith-tinydns-1.0.0/root/etc/e-smith/templates/var/service/tinydns/root/data/65dhcpARecords mezzanine_patched_e-smith-tinydns-1.0.0/root/etc/e-smith/templates/var/service/tinydns/root/data/65dhcpARecords
--- e-smith-tinydns-1.0.0/root/etc/e-smith/templates/var/service/tinydns/root/data/65dhcpARecords	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_e-smith-tinydns-1.0.0/root/etc/e-smith/templates/var/service/tinydns/root/data/65dhcpARecords	2007-01-27 20:55:56.000000000 -0700
@@ -0,0 +1,34 @@
+{
+    $OUT .= "# A Records for dhcp leases\n";
+    if(open(LEASEFILE, "/var/lib/dhcp/dhcpd.leases")) {
+	my ($sec,$min,$hour,$mday,$mon,$year,$wday,$yday,$isdst) = gmtime(time);
+	my $curdate = sprintf "%04d%02d%02d%02d%02d%02d", ($year+1900),($mon+1),$mday,$hour,$min,$sec;
+
+	while(<LEASEFILE>) {
+	    my ($ip, $hostname, $mac, $enddate, $endtime);
+
+	    if (/^\s*lease\s+(\S+)/i) {
+		$ip = $1;
+
+		undef $hostname;
+		undef $enddate;
+
+		while ($_ !~ /^\}/i) {
+		    $_ = <LEASEFILE>;
+		    $hostname = lc($1) if ($_ =~ /^\s*client.*"(.*)"/i);
+		    $enddate = $1 if ($_ =~ /^\s*ends\s+\d\s+([^;]+)/i);
+		}
+		if ($hostname || $enddate) {
+		    $enddate =~ s|[/: ]||g;
+		    if ($enddate > $curdate) {
+			next if $hosts->get("$hostname.$DomainName");
+
+			$OUT .= ($allocated_ips{$ip} ? "+" : "=") . "$hostname.$DomainName:$ip\n";
+			$allocated_ips{$ip} = 1;
+		    }
+		}
+	    }
+	}
+	close (LEASEFILE);
+    }
+}
